org.openjkdf2.OpenJKDF2.yml

```diff
@@ -0,0 +1,32 @@
+app-id: org.openjkdf2.OpenJKDF2
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+
+command: openjkdf2-64
+
+modules:
+  - packaging/flatpak/glew/glew.json
+  - packaging/flatpak/glu/glu-9.json
+  - packaging/flatpak/python3-cogapp.json
+  
+  - name: openjkdf2-64
+    buildsystem: simple
+    build-commands:
+      - bash packaging/flatpak/build_linux64_flatpak.sh
+    sources:
+      - type: git
+        url: https://github.com/shinyquagsire23/OpenJKDF2
+        commit: 421a3d927e756ad47820893aea0bfacb6e749772
+
+cleanup-commands:
+  - 'pip3 uninstall -y cogapp' # We only use this for building
+
+finish-args:
+  - --device=all # needed for gamepads
+  - --share=ipc  # needed for X11
+  - --share=network # needed for networking (Multiplayer/Valve GameNetworkingSockets)
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --filesystem=~/.local/share/openjkdf2:rw # Assets, mods, and saves are located here.
+  - --filesystem=host:ro # One-time asset installation prompts user for a disk (ie /run/media mounts) or an existing install (ie Steam, GOG).
```

Comment:
this is too broad.

Reply:
Appreciate the pushback on this, I moved to GtkFileChooserNative and the sandboxing had no issues.